### PR TITLE
Skip debug file with case debug also stripped

### PR DESCRIPTION
**Pull request by shaopeng-gh**

### CODE REVIEW

File: src/BinaryParsers/ElfBinary/ElfBinary.cs [hunk 1]
```diff
@@ -305,6 +308,15 @@ private bool SectionExistsAndHasNoBits(string sectionName)
             return sectionRetrieved != null && sectionRetrieved.Type == SectionType.NoBits;
         }
 
+        /// <summary>
+        /// Get if section not exists or also exists but has no bits
+        /// </summary>
+        private bool SectionNotExistsOrHasNoBits(string sectionName)
```
Comment (eddynaka) [inline src/BinaryParsers/ElfBinary/ElfBinary.cs:314]: SectionDoesNotExist...

Comment (eddynaka) [reply src/BinaryParsers/ElfBinary/ElfBinary.cs:314]: also, change the summary

Comment (shaopeng-gh) [reply src/BinaryParsers/ElfBinary/ElfBinary.cs]: fixed

File: src/BinaryParsers/ElfBinary/ElfBinary.cs [hunk 2]
```diff
@@ -199,7 +199,10 @@ public List<SymbolEntry<ulong>> GetSymbolTableFiles()
         /// <value>
         ///   <c>true</c> if is debug only file; otherwise, <c>false</c>.
         /// </value>
-        public bool IsDebugOnlyFile => DebugFileType == DebugFileType.DebugOnlyFileDebuglink || DebugFileType == DebugFileType.DebugOnlyFileDwo;
+        public bool IsDebugOnlyFile => DebugFileType ==
```
Comment (eddynaka) [inline src/BinaryParsers/ElfBinary/ElfBinary.cs:202]: do we have a sample file to add a test?

Comment (shaopeng-gh) [reply src/BinaryParsers/ElfBinary/ElfBinary.cs:202]: added test file and test

**Review by eddynaka — APPROVED** [review]

**Pull request merged**